vertical-pod-autoscaler/deploy/vpa-rbac.yaml

```diff
@@ -4,6 +4,20 @@ kind: ClusterRole
 metadata:
   name: system:metrics-reader
 rules:
+  - apiGroups:
```

Comment:
We need a release note about changing permissions.

Also I expect many VPA instances will not need those permissions. What do you think about not giving those permissions unless necessary

Reply:
I thought that the `vertical-pod-autoscaler/deploy/*` config were all just to support testing.  The actual deployment configurations should be under the top-level `charts/` subdirectory.  Is that wrong?

Reply:
I think leaving the permissions out unless necessary is the right thing to do.  You can't configure a recommender with external metrics without doing some of your own configuration (e.g., flags for the names of the metrics to use for cpu and memory).  I think when mentioning external metrics support in the release notes, we should also mention the flags and additional permissions required.

Reply:
I've reduced them to just `externalmetrics` now, and will move them over now to another file.